assets/src/edit-story/components/button/index.js

```diff
@@ -30,6 +30,7 @@ import {
   Redo as RedoIcon,
   ArrowLeft as ArrowLeftIcon,
   ArrowRight as ArrowRightIcon,
+  ArrowDown,
```

Comment:
[nit]: Should this be aliased as well, since the others also are? For consistency.

Reply:
Done.